packages/odyssey-react-mui/src/labs/DateField.tsx

```diff
@@ -47,17 +72,144 @@ const DateField = ({
   hasInitialFocus,
   hint,
   id: idOverride,
+  inputRef,
   isDisabled = false,
   isOptional = false,
   isReadOnly,
   label,
+  minDate,
+  maxDate,
   onBlur,
   onChange,
   onFocus,
+  timezone,
   value,
 }: DateFieldProps) => {
+  const errorMap = useOdysseyDateError();
+  const [displayedErrorMessage, setDisplayedErrorMessage] = useState<
+    string | undefined
+  >(errorMessage);
```

Comment:
Is the type of `errorMessage` also `string | undefined`? If so, you can remove it from the generic as TS can infer that type:
```suggestion
  const [displayedErrorMessage, setDisplayedErrorMessage] = useState(errorMessage);
```